src/OWML.ModHelper.Assets/ModAsset.cs

```diff
@@ -20,7 +20,7 @@ public void SetAsset(T asset)
             Loaded?.Invoke(asset);
         }
 
-        private void Start()
+        public void Start()
```

Comment:
🤔 why would Start ever need to be public?

Reply:
I bet it's because code analysis complains that the private method is unused :P

Reply:
Correct! Also, I've accidentally deleted code like that in the past because the IDE says it's not in use :)

Reply:
Actual Unity projects don't seem to have this problem. Might be useful to check what's special about an sln project generated by Unity, compared to one made manually.